### ISSUE TITLE
SlevomatCodingStandard.Namespaces.UnusedUses false positive on @method annotation

### ISSUE THREAD

**Issue by watari** [slevomat/coding-standard#661]
Hi there!
I encountered false positive `SlevomatCodingStandard.Namespaces.UnusedUses` sniff warning when class is used in `@method` annotation:

         <?php
         use App\Constant\Fallback;
         
         /**
          * Class Dimension
          * @codingStandardsIgnoreStart
          * @method DimensionL10n|null getLocalizedModel(string $langId = null, bool $createIfEmpty = false, string $collectionAttribute = null, string $fallback = Fallback::NONE)
          * @codingStandardsIgnoreEnd
          */
         class Dimension implements DimensionInterface
         {
         }

Problem present even when annotations `@codingStandardsIgnoreStart` and `@codingStandardsIgnoreEnd` is removed. 

Sniff config is next:

         <rule ref="SlevomatCodingStandard.Namespaces.UnusedUses">
                 <properties>
                     <property name="searchAnnotations" value="1" />
                 </properties>
         </rule>

Is there present any bypass?

Version 4.8.7
 Warning message is next - `Type App\Constant\Fallback is not used in this file.`

**Comment by kukulich** [comment]
@watari What class is reported as unused?

**Comment by watari** [comment]
@kukulich , `Sorry, I forgot about class (:
App\Constant\Fallback`. Warning message is next - `Type App\Constant\Fallback is not used in this file.`

**Comment by watari** [comment]
Version 4.8.7

**Comment by kukulich** [comment]
@watari Version 4.8.7 doesn't not suport this.

**Comment by watari** [comment]
Interesting. Sniff worked for me on this version. Trouble was only with reported case. I checked version in `composer.lock` of my app. 
I update my version to the 5.* and check again

**Comment by watari** [comment]
@kukulich . BTW, in 4.8.6 present next message - `UnusedUsesSniff: Fixed false positive`
[changelog for 4.8.6](https://github.com/slevomat/coding-standard/releases/tag/4.8.6)

**Comment by kukulich** [comment]
@watari The sniff exists in 4.8.6 but it does not fully support `@method` annotation

**Comment by watari** [comment]
@kukulich , I updated package to version 5.0.4 and everything works fine. Thank you for fast reply.
I close my issue. Have a nice day!

**Comment by watari** [comment]
My mistake. Error still present. Composer erased to default my set of rules. version 5.0.4

**Comment by tominventisbe** [comment]
FYI, this also seems to happen on embedded `@see` tags:

```php
use DateTime;

/**
 * Foo {@see DateTime} bar.
 */
```

`DateTime` is reported as unused here.

It works fine if you use the plain `@see` tag:


```php
use DateTime;

/**
 * Foo bar.
 *
 * @see DateTime
 */
```

**Comment by kukulich** [comment]
Yes, there's no support for embeded `@see`

**Comment by kukulich** [comment]
Fixed in https://github.com/slevomat/coding-standard/commit/0be613ed1cca6aeed99c20a17e45692bb9bcfe67
